lib/navigation/model/routes/profile_path.dart

```diff
@@ -15,7 +15,20 @@ class ProfilePath extends RoutePath {
               .fetchPerson(name),
           builder: (_, snapshot) {
             if (snapshot.connectionState == ConnectionState.done) {
+              if (!snapshot.hasData) {
+                return ErrorPage(
+                  errorMessage: S.current.labelUnknown,
+                );
+              }
+
               final Person personData = snapshot.data;
+
+              if (personData.office == null || personData.email == null) {
```

Comment:
Why? If the email/office is null we should just show '-' or some empty string.

Reply:
Because `fetchPerson` return a `Person` even if it doesn't exist. I didn't mess with that logic. https://github.com/student-hub/acs-upb-mobile/blob/01940a5bb70116d820b05b28f535498f4510018a/lib/pages/people/service/person_provider.dart#L45-L46

Reply:
I see what you mean, but this doesn't seem like the behaviour we want. We may not have a person in our database but they may actually have classes registered, for instance, which we could show on the person page. I think it's okay to open the person page even if we only know a name, and no other info.

Alternatively, you can create a method on Person like hasInterestingInfo which checks that we have at least one relevant field we can show other than the name (email, office, position etc, maybe even classes). But I think not having this check at all is easier and a reasonable behaviour.

Reply:
ok, linking a new pr for that in a minute

Reply:
https://github.com/student-hub/acs-upb-mobile/pull/305